tasks/kernel_matrix_testing/kmt_os.py

```diff
@@ -65,6 +65,9 @@ def init_local(ctx):
         ctx.run(f"{sudo} sed --in-place 's/#user = \"root\"/user = \"{user}\"/' {Linux.qemu_conf}")
         ctx.run(f"{sudo} sed --in-place 's/#group = \"root\"/group = \"kvm\"/' {Linux.qemu_conf}")
 
+        ctx.run("{sudo} echo \"/opt/kernel-version-testing 100.0.0.0/8(ro,no_root_squash,no_subtree_check)\" >> /etc/exports")
```

Comment:
Two things:
- You're missing the `f` at the beginning of the string here
- Should we check the file to ensure we don't add duplicate entries if someone calls init multiple times?

Reply:
Forgot to push the latest commit before opening the PR